Grupp 2 Assignment Done!

msu999 opened the pull request:
If the tex-file does not compile then our work can be viewed on overleaf right here:
https://www.overleaf.com/3792792543mxmdnyfghcwy#ea6978

msu999 committed:
Add files via upload

msu999 committed:
Add files via upload

vagdur committed:
fix overwide figure

Your image file had a big white margin, so you needed to make it too wide (an overfull hbox error) to make the text readable - cropping out the whitespace makes the spacing behave better, and we can pick a \textwidth multiplier that doesn't overflow

vagdur committed:
crop excess whitespace in other images

While I was at it, did this tweak here too - but they seem to not be used in your actual TeX?

vagdur committed:
original exercise solutions

I copy-pasted this right from your Overleaf - it doesn't compile on my machine, but hopefully that can be fixed

vagdur committed:
remove double newlines

This causes my TeX compiler to throw an error.

vagdur committed:
add in three forgotten figures

These were missing in your PR, but I found them on your Overleaf and added them in

vagdur committed:
fix graphics locations

We can't have a folder named "Exercise3" sitting in the main graphics folder - there's an exercise three in every lecture, so the name is ambiguous?

vagdur committed:
cropped whitespace from figures

vagdur committed:
remove unused images

vagdur committed:
footnote to sidenote

Footnotes are turned into sidenotes in this style anyway, but a \footnote will throw a warning

vagdur committed:
remove unused duplicate label

You already labelled a previous figure as "fig:G" - and don't actually refer to either figure by its label. So I just deleted the label from one of them

vagdur commented:
I have now pushed a bunch of changes here to add in your exercise solutions, and to resolve a few error messages (Overleaf is a bit more lenient on what it will allow without error messages than my machine is) and stylistic things. Have a look through what I changed if you want to polish your LaTeX skills.

Also, please change the "Authors" bit in the exercises from "Group 2" to the actual names of the people who contributed.

(I haven't looked at the actual mathematical content of your work yet - I'll do that later. This was just a first pass through to make sure it actually compiles so it is possible for me to read.)

msu999 committed:
Update lecture6_solutions2.tex

Names of group members added

msu999 committed:
Update lecture6_solutions2.tex

Removed redundant sidenote from ex. 6 and clarified it.

msu999 committed:
Update lecture6.tex

Removed redundant sidenote from ex. 6 and clarified it.

vagdur committed:
remove remaining unused images

vagdur commented:
[lecture6 group 2.pdf](https://github.com/vagdur/Grafteori-1MA170/files/13579770/lecture6.group.2.pdf)
[lecture6_solutions2 group2.pdf](https://github.com/vagdur/Grafteori-1MA170/files/13579772/lecture6_solutions2.group2.pdf)

msu999 committed:
Update of solution sheet after feedback

msu999 committed:
Update lecture notes after feedback

msu999 commented:
@vagdur We are done with updating our work based on your feedback.

vagdur committed:
Merge branch 'main' into pr/2

vagdur committed:
moved graphic into folder, fix path in TeX

vagdur committed:
remove double linebreaks

These don't work in my local TeX engine, and are bad practice anyway

vagdur committed:
minor typo fixes and indentation fixes in code

vagdur committed:
consistent big-O notation

I believe the mathcal big O is from computer science, maybe? In maths papers (and all the other lectures) it is just a normal font O, so let's keep it that way here

vagdur committed:
fix oversized figure and resulting sidenote placements

the sidenotes started to overlap when things moved because the oversized graphic changed size, so these sidenote tweaks go in the same commit as that intentionally

vagdur committed:
undo Kruskal's algorithm re-definition

after some thought, I think the original version is preferable, so I am undoing this change

vagdur committed:
typo fix

vagdur committed:
fix erroneous \neq

Upon rereading this, I realize that the original = was correct, so reverting this

vagdur committed:
remove new section

There's nothing per se wrong with this section - everything it says is true - but I am not sure I want to be spending the time on saying this in the lectures

vagdur committed:
tweak sidenote placement

this doesn't need to be shifted any longer, since the figure whose label it was shifted to accommodate is no longer there

vagdur committed:
incorporate change from other group

the other group that worked on this lecture made some changes that need to be merged into your changes

vagdur committed:
language tweak

vagdur committed:
add back exercises from other group's submission

vagdur committed:
add back in figure from other group's submission

vagdur committed:
fix caption for xca figure

vagdur committed:
add xca solutions link to course page

vagdur committed:
language tweak in other group's xca

The given graph does not have all weights distinct, so a priori the MST might not be unique
